.gitignore

```diff
@@ -128,6 +128,7 @@ collectors/python.d.plugin/python.d.plugin
 collectors/fping.plugin/fping.plugin
 collectors/ioping.plugin/ioping.plugin
 collectors/go.d.plugin
+web/netdata-switch-dashboard.sh
 
 # installer generated files
 /netdata-uninstaller.sh
```

Comment:
I also see that scrolling is very slow compared to the old version. I checked for CPU and Mem usage and didn't see anything signficantly different vs the old dashboard.

Reply:
I also saw that the scrolling is worse than on old dashboard. On Mac it was looking ok, because there are not so many charts, but on Linux the difference was clear. I've done few tweaks to address that (at the cost of code cleanliness), and i think  the difference is about 5 times smaller than before.

The fixes are under review now - https://github.com/netdata/dashboard/pull/36
I'll update that branch soon, after it's approved

I haven't yet fixed 2 issues submitted by @vlvkobal and @mfundul - redirect from alarms/log, and going to `master` from `slave` streaming node. I'll fix them as soon as i'll have time, probably today evening.